config/routes.rb

```diff
@@ -520,6 +520,7 @@
 
   scope controller: 'sys' do
     match '/sys/projects.:format', action: 'projects', via: :get
+    match '/sys/projects/:id/repository/storage', action: 'update_required_storage', via: :get
```

Comment:
Only talking about naming here, but since this is an endpoint that does not return anything and is only meant to be pushed, I would vote for something with a verb, e.g. `update_storage`, rather than `storage`